src/internal-version.js

```diff
@@ -24,5 +24,5 @@
  * @return {string}
  */
 export function internalRuntimeVersion() {
-  return '$internalRuntimeVersion$';
```

Comment:
For consistency, would it make sense to also remove the `'$internalRuntimeVersion$'` sentinel from all the call-sites in unit tests, etc. and replace it with `INTERNAL_RUNTIME_VERSION`?

https://github.com/ampproject/amphtml/search?q=%22%24internalRuntimeVersion%24%22

This might make it easier for a new engineer to search through the code while writing / debugging tests.

If you think it's worth it, doesn't have to be a part of this PR.

Reply:
Are you proposing we also do the var replacement at unit test time? If so, I agree. Seems like `'$internalRuntimeVersion$'` is purely vestigial

Reply:
Exactly what I was thinking. IIRC, the only reason test code uses `'$internalRuntimeVersion$'` is because `src/internal-version.js` was using it. The sentinel can be anything we want it to be.

Reply:
I don't think we need a sentinel. we could do the actual var replacement for unit tests

Reply:
Ok great. Will pursue this in a followup, since I can then do other related work in parallel